### PR TITLE
62 single skill volunteer

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#62

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -91,6 +91,16 @@ function App() {
                 allowedRoles={[roles.HQ]}
                 layout="withoutHeader"
               />
+              {/* VOLUNTEER ROUTES */}
+              <Route
+                exact
+                path={navRoutes.VOLUNTEER.INDIVIDUAL_SKILL}
+                Component={Pages.VolunteerIndividualSkill}
+                isPrivate
+                allowedRoles={[roles.VOLUNTEER]}
```
Comment (thejoefriel) [inline client/src/App.js:100]: - [ ] I can't seem to access the skills unless logged in as a volunteer? They should be publicly available but if you're not logged in then you see this version of the page instead: https://www.figma.com/file/9oFoQawgjFYwHi1dG77RH7/The-Scouts-Association-MVP?node-id=1449%3A5804

File: client/src/components/Card/ActivityCard/style.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+import { Link } from 'react-router-dom';
+
+export const CardWrapper = styled(Link)`
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.03), 0px 1px 8px rgba(0, 0, 0, 0.05),
```
Comment (thejoefriel) [inline client/src/components/Card/ActivityCard/style.js:5]: use the theme shadow

File: client/src/components/Icon/icons/ArrowDown.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    viewBox="0 0 24 24"
+    width={width}
+    height={height}
+    transform={`rotate(${rotate})`}
+    {...props}
+  >
+    <path
+      d="M3.57703 7.62229L12.2835 15.9304C12.3316 15.9763 12.4095 15.9763 12.4575 15.9304L21.1641 7.62229"
+      stroke={color || '#1E1E1E'}
+      stroke-width="2.4"
```
Comment (thejoefriel) [inline client/src/components/Icon/icons/ArrowDown.js:20]: change all these in icons to camel case to reduce console warnings

File: client/src/components/Icon/icons/CupWithFlash.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <path
+      d="M13.7502 36.25H26.2502M20.0002 36.25V26.25M30.0002 17.5C30.0002 13.5438 29.9939 6.98203 29.9908 5C29.9908 4.66848 29.8591 4.35054 29.6247 4.11612C29.3903 3.8817 29.0723 3.75 28.7408 3.75L11.2572 3.77031C10.9264 3.77031 10.609 3.90147 10.3747 4.13505C10.1404 4.36862 10.0083 4.68556 10.0072 5.01641C10.0072 7.40547 9.99707 15.1078 9.99707 17.5C9.99707 22.5219 16.4814 26.25 19.9971 26.25C23.5127 26.25 30.0002 22.5219 30.0002 17.5Z"
+      stroke={strokeColor}
+      stroke-width="2.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M30 7.5H36.25V8.75C36.25 13.0641 33.6289 17.5 30 17.5M10 7.5H3.75V8.75C3.75 13.0641 6.37109 17.5 10 17.5V7.5Z"
+      stroke={strokeColor}
```
Comment (thejoefriel) [inline client/src/components/Icon/icons/CupWithFlash.js:19]: same on this file as above

File: client/src/pages/VolunteerIndividualSkill/style.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  font-weight: 700;
+  font-size: 16px;
+`;
+
+export const InfoCard = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: ${({ theme }) => theme.spacings[4]};
+  background-color: ${({ theme }) => theme.colors.white};
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.03), 0px 1px 8px rgba(0, 0, 0, 0.05),
```
Comment (thejoefriel) [inline client/src/pages/VolunteerIndividualSkill/style.js:31]: use shadow from theme

**Review by thejoefriel — CHANGES_REQUESTED** [review]
- [x] Reduce spacing between you're doing great card and the cards above. Should be 64px 
image

File: client/src/components/Card/ActivityCard/style.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/components/Card/ActivityCard/style.js:5]: done

File: client/src/components/Icon/icons/ArrowDown.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/components/Icon/icons/ArrowDown.js:20]: done

File: client/src/components/Icon/icons/CupWithFlash.js (hunk 4 above)
Comment (RamyAlshurafa) [reply client/src/components/Icon/icons/CupWithFlash.js:19]: done

File: client/src/pages/VolunteerIndividualSkill/style.js (hunk 5 above)
Comment (RamyAlshurafa) [reply client/src/pages/VolunteerIndividualSkill/style.js:31]: done

File: client/src/App.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/App.js]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**